pkg/storegateway/blocks.gohtml

```diff
@@ -4,25 +4,45 @@
 <head>
     <meta charset="UTF-8">
     <title>Store-gateway: bucket tenant blocks</title>
+    <script src="https://unpkg.com/htmx.org"></script>
 </head>
 <body>
 <h1>Store-gateway: bucket tenant blocks</h1>
 <p>Current time: {{ .Now }}</p>
 <p>Showing blocks for tenant: <strong>{{ .Tenant }}</strong></p>
 <p>
-    <form>
-        <input type="checkbox" id="show-deleted" name="show_deleted" {{ if .ShowDeleted }} checked {{ end }}>&nbsp;<label for="show-deleted">Show Deleted</label> &nbsp;&nbsp;
-        <input type="checkbox" id="show-sources" name="show_sources" {{ if .ShowSources }} checked {{ end }}>&nbsp;<label for="show-sources">Show Sources</label> &nbsp;&nbsp;
-        <input type="checkbox" id="show-parents" name="show_parents" {{ if .ShowParents }} checked {{ end }}>&nbsp;<label for="show-parents">Show Parents</label> &nbsp;&nbsp;
+    <form id="configForm">
```

Comment:
Nit: rest of the elements in the html have dashes in their names, can this be `config-form`?